apps/admin/src/preload.ts

```diff
@@ -0,0 +1,16 @@
+import { contextBridge, ipcRenderer } from "electron";
+// import gmail from '@thecointech/tx-gmail';
+
+// Whitelist limits invokes to tx-gmail
```

Comment:
whitelist unfortunately broke the compiler, I should remove this code.

The whitelist probably isn't necessary, as invoke only works if there is an existing "handle" call on the node side anyway